modules/internal/String.chpl

```diff
@@ -151,6 +152,10 @@ module String {
     var shortData : chpl__inPlaceBuffer;
   }
 
+  record codePointIndex {
+    var cpidx     : int;
```

Comment:
Is this intended to be user-visible? If so, I'm sure we can do better than the field name `cpidx`.